app.js

```diff
@@ -0,0 +1,66 @@
+// Import the Express module
+import express from 'express';
+import swaggerJSDoc from 'swagger-jsdoc';
+import swaggerUi from 'swagger-ui-express';
+import userRoutes from './routes/v1/user.js';
+import authRoutes from "./routes/v1/authRoutes.js"
+import cors from "cors"
+
+
+
+// Create an Express application
+const app = express();
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json());
+app.use(cors())
+
```

Comment:
what does the cors part do ? don't remember coming across this

Reply:
basically it meant to make it so people can only add users from the login/register page